drivers/wifi/nxp/nxp_wifi_drv.c

```diff
@@ -322,6 +322,12 @@ int nxp_wifi_wlan_event_callback(enum wlan_event_reason reason, void *data)
 		net_if_dormant_on(g_uap.netif);
 		LOG_DBG("WLAN: UAP Stopped");
 
+		if (net_addr_pton(AF_INET, CONFIG_NXP_WIFI_SOFTAP_IP_ADDRESS, &dhcps_addr4) < 0) {
+			LOG_ERR("Invalid CONFIG_NXP_WIFI_SOFTAP_IP_ADDRESS");
+			return 0;
```

Comment:
Shouldn't we stop the dhcp server regardless of whether the address was removed or not?
If the address is is not set or is invalid, the server is not stopped which looks wrong, WDYT?

Reply:
You mean I should put `net_if_ipv4_addr_rm(g_uap.netif, &dhcps_addr4);` after `net_dhcpv4_server_stop(g_uap.netif);` of line 331?

Reply:
Hi @jukkar  I don't quite understand your question, could you please elaborate it？

Reply:
If the address is not valid, we return immediately. Should we still do the dhcp stopping?

So something like this
```
if (net_addr_pton(....) < 0) {
	LOG_ERR("Invalid CONFIG_NXP_WIFI_SOFTAP_IP_ADDRESS");
} else {
	net_if_ipv4_addr_rm(g_uap.netif, &dhcps_addr4);
}
```

Reply:
Hi @jukkar  updated, please review again.